src/runtime-rs/crates/shim/src/shim_run.rs

```diff
@@ -36,12 +37,18 @@ impl ShimExecutor {
 
     async fn do_run(&mut self) -> Result<()> {
         info!(sl!(), "start to run");
-        self.args.validate(false).context("validata")?;
+        self.args.validate(false).context("validate")?;
 
         let server_fd = get_server_fd().context("get server fd")?;
-        let mut service_manager = service::ServiceManager::new(&self.args.id, server_fd)
-            .await
-            .context("new runtime server")?;
+        let mut service_manager = service::ServiceManager::new(
+            &self.args.id,
+            &self.args.publish_binary,
+            &self.args.address,
+            &self.args.namespace,
+            server_fd,
+        )
+        .await
+        .context("new runtime server")?;
```

Comment:
`new shim server` ?

Reply:
Thanks for your review, has been updated.